SIPS/sip-34.md

```diff
@@ -0,0 +1,87 @@
+---
+sip: 34
+title: endowment:keyring.version
+status: Draft
+discussions-to: https://github.com/MetaMask/SIPs/discussions/172
+author: Charly Chevalier (@ccharly), Daniel Rocha (@danroc)
+created: 2025-04-07
+---
+
+## Abstract
+
+This proposal introduces a new field, `version`, within the existing configuration object used by the `endowment:keyring`.
```

Comment:
I wonder if we should make this more generic, and instead describe a way for any endowment, or permission in general, to specify a version.

Reply:
I'm definitely ok with that, as long as we have a way to use that version, I'm fine with this.

This would probably allow breaking changes within the endowments themselves yes!